StepicUITests/Sources/StepicUITests.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  StepicUITests.swift
+//  StepicUITests
+//
+//  Created by AKholin on 11.03.2021.
+//  Copyright © 2021 Alex Karpov. All rights reserved.
+//
+
+import XCTest
+
+class StepicUITests: XCTestCase {
+    
+
+    override func setUpWithError() throws {
+        // This method is called before the invocation of each test method in the class.
+        continueAfterFailure = false
+        XCUIApplication().launchArguments += ["-AppleLanguages", "(en)"]
+        XCUIApplication().launchArguments += ["-AppleLocale", "en_EN"]
+    }
+
+    override func tearDownWithError() throws {
```

Comment:
Очистка состояния не может бросить ошибку, поэтому можно использовать обычный метод `tearDown()`
```suggestion
    override func tearDown() {
```